src/goto-symex/goto_state.h

```diff
@@ -63,6 +63,13 @@ class goto_statet
   /// Threads
   unsigned atomic_section_id = 0;
 
+  /// Get the currently calculated complexity for this state. Used
+  /// to short-circuit states if they have become too computationally complex.
+  inline std::size_t get_complexity()
```

Comment:
`get_` is misleading since you would expect it to return the value of a field. `complexity()` is fine